docker-compose.yml

```diff
@@ -0,0 +1,9 @@
+version: '3.8'
```

Comment:
This file configures what happens when the developer runs `docker-compose.yml`. At the moment all it does is start up a database service, running postgres 12.3.